### ISSUE TITLE
Paste nsec to log in

### ISSUE THREAD

**Issue by chr15m** [vitorpamplona/sheetstr#1]
I would like to be able to generate a throwaway account by generating my own nsec and logging in rather than using a service to manage my keys.

**Comment by vitorpamplona** [comment]
You can log in with the extension or a NIP-46 bunker (Amber on Android, for instance). 

Nsecs on the browser are not safe at all :(

**Comment by chr15m** [comment]
Yes. As a user I accept the risks (which are zero if I am creating a throwaway).
